Quadriga/src/main/java/edu/asu/spring/quadriga/service/network/impl/NetworkTransformationManager.java

```diff
@@ -133,20 +198,9 @@ public ITransformedNetwork getTransformedNetworkOfProject(String projectId)
         // return new network with updated nodes and links
         return new TransformedNetwork(updatedNodes, links);
     }
-
-    @Override
-    public ITransformedNetwork getSearchTransformedNetwork(String projectId, String conceptId)
-        throws QuadrigaStorageException {
-        // get the transformed network and search for the concept id
-        List<INetwork> networkList = getNetworkList(projectId);
-
-        if (networkList == null) {
-            return null;
-        }
-
-        // get the transformed network of all the networks in a project
-        ITransformedNetwork transformedNetwork = getTransformedNetworkOfProject(projectId);
-
+    
+    public ITransformedNetwork getfinalTransformedNetwork(ITransformedNetwork transformedNetwork, String conceptId)
```

Comment:
needs proper camel case. also, what do you mean by 'final'? What does this method do with the provided network? And is this a method that is meant to be used by other classes as well? If not, it should be private.